### PR TITLE
feat(components.converter): add units for binary system

### PR DESCRIPTION
**Pull request by marie-j**
### Description of the Change

Allow to convert values based on octets binary system

### Related to 
https://github.com/ovh-ux/ovh-module-exchange/pull/194

MBP-7

### CODE REVIEW

File: client/app/webUniverseComponents/converter/converter.factory.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+          val: binaryBase ** 6,
+          unit: 'EiB',
+        },
+        {
+          val: binaryBase ** 7,
+          unit: 'ZiB',
+        },
+        {
+          val: binaryBase ** 8,
+          unit: 'YB',
```
Comment (FredericEspiau) [inline client/app/webUniverseComponents/converter/converter.factory.js]:
```suggestion
          unit: 'YiB',
```

?

Comment (marie-j) [reply client/app/webUniverseComponents/converter/converter.factory.js]: Nice catch 👍

**Comment by antleblanc** [comment]
Is it linked to ovh-ux/ovh-module-exchange#194?

**Comment by ovh-cds** [comment]
CDS Report build-manager#460.0 ✘  
* Stage 1
  * Build dependencies and inject Sentry ✔ 
* Stage 2
  * Build manager CA ✔ 
  * Build manager EU ✘  
  * Build manager US ✔ 
  * Tests ✔

**Pull request merged**